specs/_template/02_user_interface_chrome.rst

```diff
@@ -0,0 +1,32 @@
+=======  ==================  ======  ==========
+ epic          title         author   created
+=======  ==================  ======  ==========
+`EPIC`_  SPECIFICATION_NAME  AUTHOR  CREATED_AT
+=======  ==================  ======  ==========
+
+.. _EPIC: gnosis/safe#EPIC
+
+===========================
+User interface (Chrome)
+===========================
+
+.. contents:: Table of Contents
+
+1. SCREEN_NAME
+------------------
+
+Short description of the screen or its state
+
+.. we use raw html because otherwise GitHub doesn't change the width or height of an image.
+
+.. raw:: html
+
+   <img src="screens/chrome/ScreenName.png" width="320px"/>
```

Comment:
while it is nice to have a preview here (and a version that will not change without notice) I think it would be nice to at least also have a link to zeplin here (and maybe also invision)

Reply:
In the past, Zeplin links changed quite often. Since the design team was changing and reuploading screens. If we add Zeplin links, we should prevent this.

Reply:
That's why I think we should start without the Zeplin links actually.

Reply:
for the finals specs that are then used for implementation I would like to have a zepelin link, since that is the place we pull the resources from.

Reply:
and so, include or not? I would have it, but maybe in the epic ticket

Reply:
If you both want to have the links. Fine with me. Let's all work though towards keeping them up to date. I would add them to the specs, not only the subepics.

Reply:
Sure. under what influences does a zeplin link change?

Reply:
I would like to include the links to have a source for the client assets. If it is required to change these links we should probably update them (or just remove them if everything is done)

Reply:
Yes, let's add them Zeplin links 👌